.github/workflows/README.md

```diff
@@ -0,0 +1,28 @@
+## Workflows Naming Convention
+
+Whenever possible, follow this naming convention for workflows:
+
+```yaml
+# Use a short descriptive name for the whole workflow
+name: Validate PR Labels
+on: ...
+
+jobs:
+  # Use a present-tense verb as the name for the job, aligned with the workflow name
+  validate:
+    steps:
+      # Omit names for steps using an action
```

Comment:
Interesting! Why is this one important?

Reply:
Good question, I should have added my rationale. To me, it seems unnecessary to add a custom `name` for something that already has one. Third party actions already have a name, like `actions/checkout` or `danger/danger-js`.

Having said all that, I can see the case for a custom `name` when the action takes arguments and the value of those arguments is useful to know when scanning the workflow log, where with "workflow log" I mean this

<img width="332" alt="Screen Shot 2020-02-21 at 20 31 02" src="https://user-images.githubusercontent.com/1218433/75021809-1e425c00-54e9-11ea-9fd9-694ef991c691.png">